Add profiles in JSON and enum for the colors

Contarkos opened the pull request:
The color profiles are in JSON data arrays and are loaded as needed when the color settings is changed.

ludw commented:
I've done some preliminary testing and it does seem to work well, memory is also reduced.
Do you need the label_texts.json file for this PR though? Feels like unrelated changes.

I'd also suggest renaming propColorValues and propNightColorValues as those are not application properties (settings). Maybe just drop the prop prefix.

Contarkos commented:
I did as you suggested, deleted the unnecessary file and renamed the variables to match what they represent. I also fixed the few lines you added today with the new syntax.

ludw commented:
Thanks. Just looking from my phone it looks good. I'll take a closer look and likely merge tomorrow.
